### PR TITLE
Oracle partner highlight

### PR DESCRIPTION
**Pull request by dtaivpp**
### Description
Adds the Oracle Cloud Infrastructure Search Service Partner Highlight
 
### Issues Resolved
n/a

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+During the beta customers were able to easily migrate the Elasticsearch infrastructure to OpenSearch. Because their private endpoints use the same APIs as Elasticsearch, it was simply a matter of ingesting data and pointing to the new endpoints. In fact, one of their customers ingested 300 million items into their OpenSearch cluster and is now enabled for analysis and action with OpenSearch Dashboards.
+
+It’s a good day for open source and the OpenSearch community as another hyperscale cloud provider embraces OpenSearch. Congrats again. To learn more about OCI Search Service with OpenSearch go [here](http://oracle.com/cloud/search). 
+
+## Get involved!
+We would love to see you contribute to OpenSearch (if you’re not already)! For almost any type of contribution, the first step is opening an GitHub issue (https://github.com/opensearch-project/OpenSearch). Even if you think you already know what the solution is, writing a description of the problem you’re trying to solve will help everyone get context when they review your pull request.
```
Comment (ahopp) [inline _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown:27]: I think we should hyperlink in the [typical](google.com) form. Feels a bit cleaner in most cases.

File: _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "Partner Highlight: Oracle Cloud Infrastructure Search Service with OpenSearch is GA!"
+authors:
+  - andhopp
```
Comment (ahopp) [inline _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown:5]: Let's do Andrew, David, and Vacha!

File: _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  - vachshah
+  - dtaivpp
+date: 2022-05-10
+categories:
+  - partners
+
+excerpt: "Oracle has released it's Cloud Infrastructure Search Service!"
```
Comment (ahopp) [inline _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown:12]: Let's add "...with OpenSearch" in the excerpt.

**Review by ahopp — CHANGES_REQUESTED** [review]
Few minor tweaks but otherwise looks good! @VachaShah do you want to take a look?

File: _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+During the beta customers were able to easily migrate the Elasticsearch infrastructure to OpenSearch. Because their private endpoints use the same APIs as Elasticsearch, it was simply a matter of ingesting data and pointing to the new endpoints. In fact, one of their customers ingested 300 million items into their OpenSearch cluster and is now enabled for analysis and action with OpenSearch Dashboards.
+
+It’s a good day for open source and the OpenSearch community as another hyperscale cloud provider embraces OpenSearch. Congrats again. To learn more about OCI Search Service with OpenSearch go [here](http://oracle.com/cloud/search). 
+
+## Get involved!
+We would love to see you contribute to OpenSearch (if you’re not already)! For almost any type of contribution, the first step is opening an GitHub issue (https://github.com/opensearch-project/OpenSearch). Even if you think you already know what the solution is, writing a description of the problem you’re trying to solve will help everyone get context when they review your pull request.
```
[the remark this answers is not in the document]
Comment (dtaivpp) [reply _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown]: Absolutely, just updated!

File: _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown (hunk 3 above)
Comment (dtaivpp) [reply _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown]: Added

File: _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown (hunk 2 above)
Comment (dtaivpp) [reply _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown:5]: Do you mean rearrange the ordering? I have all three in there.

**Review by VachaShah — APPROVED** [review]
LGTM!

**Review by ahopp — APPROVED** [review]

File: _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+Oracle’s mission is "to help people see data in new ways, discover insights, unlock endless possibilities" and delivering this cloud-based search service with OpenSearch is inline with that mission. Their Gen 2 cloud is a strong foundation with its built-in security to mitigate threats, improved automation, scalability, and high performance.
+
+During the beta customers were able to easily migrate the Elasticsearch infrastructure to OpenSearch. Because their private endpoints use the same APIs as Elasticsearch, it was simply a matter of ingesting data and pointing to the new endpoints. In fact, one of their customers ingested 300 million items into their OpenSearch cluster and is now enabled for analysis and action with OpenSearch Dashboards.
+
+It’s a good day for open source and the OpenSearch community as another hyperscale cloud provider embraces OpenSearch. Congrats again. To learn more about OCI Search Service with OpenSearch go [here](http://oracle.com/cloud/search).
```
Comment (krisfreedain) [inline _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown:24]: @dtaivpp - can you change http to https?  the http threw an occasional error for me.

**Review by krisfreedain — CHANGES_REQUESTED** [review]
quick change @dtaivpp

File: _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown (hunk 5 above)
Comment (dtaivpp) [reply _posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown]: Resolved!

**Review by krisfreedain — APPROVED** [review]
looks good!

**Pull request merged**